src/ui/locales/id.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Alihkan atribusi",
+    "AttributionControl.MapFeedback": "Umpan balik peta",
+    "FullscreenControl.Enter": "Memasuki layar penuh",
+    "FullscreenControl.Exit": "Keluar dari layar penuh",
+    "GeolocateControl.FindMyLocation": "Temukan lokasi saya",
+    "GeolocateControl.LocationNotAvailable": "Lokasi tidak tersedia",
+    "LogoControl.Title": "logo kotak peta",
```

Comment:
I assume that Mapbox is a brand name so it doesn't have to be translated
```suggestion
    "LogoControl.Title": "logo Mapbox",
```